src/components/explore/ProportionExplorerView.svelte

```diff
@@ -95,12 +92,37 @@ h2 {
 
     <div class=body-control-set>
       <label class=body-control-set--label>Categories</label>
+      {#if bucketOptions.length > maxVisibleCategories}
+        <OptionMenu multi on:selection={(evt) => {
+          dispatch('selection', { selection: evt.detail.keys, type: 'activeBuckets' });
+        }}>
+          {#each bucketOptions as bucketOption, i (bucketOption)}
+            <Option
+              selected={activeBuckets.includes(bucketOption)}
+              key={bucketOption}
+              label={bucketOption}>
+              <!--
```

Comment:
this note feels unnecessary in the source. This is a known design choice in svelte at this juncture and we follow this pattern throughout GLAM and graph-paper already